### PR TITLE
Wording

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: cafebabel/templates/articles/proposals/create.html [hunk 1]
```diff
@@ -42,11 +46,11 @@ <h3>Wanna be part of Cafébabel project? That’s the right place</h3>
           </select>
         </label>
       </div>
-    <label for=field_additional>Additional informations</label><textarea name=additional id=field_additional></textarea>
+    <label for=field_additional>Additional information</label><textarea name=additional id=field_additional></textarea>
 
-    <button type=submit>Suggest this article</button>
+    <button type=submit>You're all done! Submit your pitch</button>
```
Comment (davidbgk) [inline cafebabel/templates/articles/proposals/create.html]: I'd use a `’` instead of the `'` to be consistent.

**Review by davidbgk — APPROVED** [review]
r+wc

File: cafebabel/templates/articles/proposals/create.html (hunk 1 above)
Comment (johangiraud) [reply cafebabel/templates/articles/proposals/create.html]: ok !

File: cafebabel/tests/test_articles.py [hunk 2]
```diff
@@ -370,10 +370,10 @@ def test_article_to_translate_should_have_translation_links(
     article.modify(language=language)
     response = client.get(f'/en/article/to-translate/?from=fr&to=en')
     assert (f'href="/en/article/translation/new/'
-            f'?lang=en&original={article.id}">Translate in English</a>'
+            f'?lang=en&original={article.id}">Translate into English</a>'
```
Comment (johangiraud) [inline cafebabel/tests/test_articles.py]: Sorry @davidbgk my page hadn't been refreshed, so I fixed the tests a second time :-(

File: cafebabel/templates/articles/translations/create.html [hunk 3]
```diff
@@ -16,18 +16,18 @@
     {% endif %}
     <h1 class="edit active">
       <label for=title>Title</label>
-      <input id=title name=title required placeholder="Translation in {{ language }} for article “{{ article.title }}”">
+      <input id=title name=title required placeholder="{{ article.title }}">
```
Comment (davidbgk) [inline cafebabel/templates/articles/translations/create.html:19]: I'm getting back at this 🙈 

To me a placeholder is to give an example of what could be typed, here I think the title of the original article is valuable to stay during edition to be able to translate it while seeing it. Proposition: putting it within the `label` like that `<label for=title>Title (original: {{ article.title }})</label>` maybe even with a link to the article in order to be able to access it quickly given there is no text to translate on that page.

Comment (johangiraud) [reply cafebabel/templates/articles/translations/create.html:19]: hum it seems to me more confusing to have the original title as a `label`. I will leave it like that for now, and wait for feedback from users. But thanks for your proposition 😉

**Comment by johangiraud** [comment]
@davidbgk il reste juste un cas de figure où on a la phrase _Your article was successfully updated._ c'est quand on a un article publié et qu'on le passe en draft. Désolé d'être picky mais tu peux changer ça please ?

**Comment by davidbgk** [comment]
@cafebabel-webmaster I don't get it, there is no occurrence of that string in the code

**Comment by davidbgk** [comment]
@cafebabel-webmaster and we have no option to give that particular message on status change (as previously discussed)

**Comment by johangiraud** [comment]
hum yes, that's why I was wondering... strange

EDIT : OK it's working, my fault, I hadn't pulled on the right branch

**Pull request merged**